src/System.Security.Cryptography.Xml/tests/RSAKeyValueTest.cs

```diff
@@ -82,6 +82,22 @@ public void LoadXml_LoadXml_GetXml()
         }
 
         [Fact]
+        public void LoadXml_GetXml_With_NS_Prefix()
```

Comment:
I believe this will need `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]` since this change would need to be ported to work on full framework